<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterDialog</class>
 <widget class="QDialog" name="FilterDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icon/icons/filter_icon.png</normaloff>:/icon/icons/filter_icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,0">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OkBtn">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="CancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>10</number>
     </property>
     <item row="0" column="2">
      <widget class="QLineEdit" name="LastNameLE">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Криворотов</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="LastNameL">
       <property name="text">
        <string>Фамилия: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="ZodiacSignL">
       <property name="text">
        <string>Знак зодиака: </string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDateEdit" name="BirthDayDE">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="keyboardTracking">
        <bool>true</bool>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="maximumDate">
        <date>
         <year>2023</year>
         <month>12</month>
         <day>31</day>
        </date>
       </property>
       <property name="minimumDate">
        <date>
         <year>1922</year>
         <month>12</month>
         <day>31</day>
        </date>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::DaySection</enum>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2001</year>
         <month>9</month>
         <day>23</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="BirthDayL">
       <property name="text">
        <string>Дата рождения: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="LastNameCB">
       <property name="text">
        <string/>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="ZodiacSignCB">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="BirthDayCB">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="ZodiacSignCB_2">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="maxVisibleItems">
        <number>12</number>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Дева</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Близнецы</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Весы</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Водолей</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Козерог</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Лев</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Овен</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Рак</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Рыбы</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Скорпион</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Стрелец</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Телец</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
